build.gradle

```diff
@@ -28,11 +28,18 @@ dependencies {
 // See https://github.com/JetBrains/gradle-intellij-plugin/
 intellij {
     version '2019.1.3'
-    updateSinceUntilBuild=false
+    updateSinceUntilBuild = false
+    plugins = [
```

Comment:
Is this dependency needed for the new features?

Reply:
It actually never occurred to me to try without, since it is so useful overall. Disabling or removing the `jbehave-support-plugin` will in fact make it so that these new markers do not appear. The plugin overall still works and the right-click actions are unaffected.